screens/OrdersScreen.js

```diff
@@ -50,6 +49,37 @@ class OrdersScreen extends React.Component {
 
   componentDidMount() {
     this.props.getOrdersByDateRange()
+    this._searchOrder = this.props.navigation.addListener('focus', () => {
+      if (this.state?.searchFilter?.searchTypeIndex === 0) {
+        // const shiftId = this.props.navigation.getParam('shiftId')
+        const shiftId = this.props.route.params?.shiftId
+        const dateRangeToUse = shiftId != null ? 0 : this.state.searchFilter.dateRange
+
+        this.handleOrderSearch({
+          dateRange: dateRangeToUse,
+          shiftId: shiftId,
+          fromDate: this.state.searchFilter.fromDate,
+          toDate: this.state.searchFilter.toDate,
+          searchTypeIndex: this.state.searchFilter.searchTypeIndex,
+          tableName: this.state.searchFilter.tableName,
+        })
+        // this.props.navigation.setParams({shiftId: undefined})
```

Comment:
Why did you comment this line out?

Reply:
It's comment with RA261. I think can remove it now.